### ISSUE TITLE
No alive backends when using Elasticsearch catalog with CN

### ISSUE THREAD

**Issue by snippins** [StarRocks/starrocks#37170]
Running StarRocks cluster on k8s with shared_data mode (only CNs)
The cluster has 1 Fe and 2 CNs
Deployment used kubectl and follows instructions from [starrocks-kubernetes-operator](https://github.com/StarRocks/starrocks-kubernetes-operator)

### Step to reproduce

```
CREATE EXTERNAL CATALOG elastic
COMMENT 'elastic'
PROPERTIES
(
    "type" = "es",
    "es.type" = "_doc",
    "hosts" = "https://XXX:9200",
    "es.net.ssl" = "true",
    "user" = "XXX",
    "password" = "YYYY",
    "es.nodes.wan.only" = "true"
);

select * from elastic.default_db.index_name
```

### Expected behavior (Required)
Return the result of the query

### Real behavior (Required)
SQL Error [1064] [HY000]: (conn=55806) No Alive backends

### StarRocks version (Required)
3.1.5-5d8438a

This issue seems similar to a resolved issue when query using file engine https://github.com/StarRocks/starrocks/issues/34514
Perhaps similar mistakes are made handling the elasticsearch catalog?

**Comment by Jin-H** [comment]
Similar with https://github.com/StarRocks/starrocks/pull/35193

**Comment by snippins** [comment]
@Jin-H it seems 3.2.0 image is not yet built? I can only find the release candidate version [3.2.0-rc01](https://hub.docker.com/layers/starrocks/cn-ubuntu/3.2.0-rc01/images/sha256-40a2e7314c7291056d2e0a7291727a05fecc4a9c52c499612437b59bf961ce86?context=explore)

**Comment by Jin-H** [comment]
@snippins You're right. maybe you need try build image yourself or You may have to wait for the official image

**Comment by Jin-H** [comment]
@kevincai Can you help build StarRocks 3.2.0 docker images and push to dockerhub, thanks.

**Comment by Jin-H** [comment]
@snippins Now you can try StarRocks 3.1.6 which fixed this issue [3.1.6](https://hub.docker.com/layers/starrocks/fe-ubuntu/3.1.6/images/sha256-ec5be0f1dc466f6a9ff14a030a4de461e93af5b3671f324e7d303a8884e0c78e?context=explore)
